website/docs/r/managed_disk.html.markdown

```diff
@@ -90,8 +90,25 @@ The following arguments are supported:
     operation targets a source that contains an operating system. Valid values are `Linux` or `Windows`
 * `disk_size_gb` - (Required) Specifies the size of the managed disk to create in gigabytes.
     If `create_option` is `Copy`, then the value must be equal to or greater than the source's size.
+* `encryption_settings` - (Optional) Species the encryption settings for the disk as documented below. 
+See [Azure Disk Encryption for Windows and Linux IaaS vms](https://docs.microsoft.com/en-us/azure/security/azure-security-disk-encryption) 
+for more information.
 * `tags` - (Optional) A mapping of tags to assign to the resource.
 
+`encryption_settings` supports the following:
+
+* `enabled` - (Required) Specifies if the encryption is enabled. If false, disk_encryption_key and key_encryption_key must not be specified.
```

Comment:
(same as the comments below for these 3 properties)

Reply:
done